content/uiux/concepts/ux-psychology/terms/jacobs-law/jacobs-law.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: "Jacob's Law"
+Description: "Users prefer your site to work like others they already know, because they spend most of their time on other sites."
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+Tags:
+  - 'Design'
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Jacob’s Law**, proposed by Jakob Nielsen, highlights an important principle in UX Design: users spend most of their time on other websites and applications, which shapes their expectations for how things should work. Rather than creating entirely unique designs or interfaces, aligning with familiar patterns and conventions leads to smoother user experiences and less cognitive friction.
```

Comment:
```suggestion
**Jacob’s Law**, proposed by Jakob Nielsen, highlights a key UX design principle: users spend most of their time on other websites and applications, which shapes their expectations for how interfaces should work. Instead of creating entirely unique designs, aligning with familiar patterns and conventions results in a smoother user experience and reduces cognitive friction.
```